src/docs/reference/core/Configuration/README.md

```diff
@@ -54,6 +59,26 @@ In addition you can specify a `Tenant` setting by using the Tenant Name, in this
 }
 ```
 
+You can also configure `IOptions` from code in the web project's `Startup` class as explained in the [ASP.NET documentation](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options). 
+
+A lot of Orchard Core features are configured through the admin UI with site settings stored in the database. If you wish to override the site settings, you can do this with your own configuration code.
+
+For example, the Email module allows SMTP configuration via the `SmtpSettings` class which by default is populated from the given tenant's site settings, as set on the admin. However, you can override the site settings from the `Startup` class like this (note that we use `PostConfigure` to override the site settings' values but if the module doesn't use site settings you can just use `Configure`):
+
+```
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddOrchardCms(builder => builder.ConfigureServices(services =>
+        services.PostConfigure<SmtpSettings>(settings =>
+        {
+            // You could e.g. fetch the configuration values from and injected IShellConfiguration instance here.
+            settings.Port = 255;
+        })));
+}
+```
+
+This will thus make the SMTP port use this configuration despite any other value defined from site settings. Note that on the admin there will be no indication that this override happened (and the value displayed there will still be the one configured in site settings) so if you choose to do this you'll need to let your users know.
```

Comment:
```suggestion
This will thus make the SMTP port use this configuration despite any other value defined from site settings. 

!!! note 
    On the admin there will be no indication that this override happened, and the value displayed there will still be 
    the one configured in site settings, so if you choose to do this you'll need to let your users know.
```

Reply:
Thanks, didn't know about this. I assume this will be parsed by MkDocs as it's not a standard MD (or GitHub-flavored MD) notation?

Reply:
I (think) it's part of the material theming it uses